types/request/request-tests.ts

```diff
@@ -626,25 +626,26 @@ request.get('http://10.255.255.1', {timeout: 1500}, (err) => {
 });
 
 const rand = Math.floor(Math.random() * 100000000).toString();
-  request(
```

Comment:
Are the changes to the tests just formatting? 
I don't see any ref to the `agentOptions` changes. 

Can you add some options to the [existing tests that cover the arg](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fb73e71ee0eb490a8b5fddabd7b3a5e623f664e5/types/request/request-tests.ts#L536-L562) so they include some keys from `https.AgentOptions` that are not present in just `tls.SecureContextOptions`. eg. `maxCachedSessions`, `keepAlive`, and `timeout`.

Reply:
It's to fix lint error about code vertical alignment. Select 'Hide whitespace changes' in 'Diff settings' you will find that there is nothing changed.

I'll add some options.